### ISSUE TITLE
Restrict suggestions to "useful" suggestions

### ISSUE THREAD

**Issue by tthurman** [openstenoproject/plover#670]
Classification: Enhancement
Reproducibility: N/A

## Summary

A setting to restrict the suggestions window to "useful" strokes.

"Useful" here is defined as taking fewer strokes than what you actually typed.

For example, suppose I type "funny" in two strokes as TPUPB/TPHEU. At present (4.0.0.dev0) the suggestions window displays:

```
 funny:
    TPAOUPB
    TPUP/KWREU
    TPUB/KWREU
    TPUPB/TPHEU
    TPUPB/KWREU
    TPUPB/KWRAOEU
```
The only one of these I might want to know is the single-stroke brief TPAOUPB.

This would also mean that fingerspelling a word would suggest the simplest strokes for it,. without the user having to look it up.

**Comment by GuiltyDolphin** [comment]
@tthurman I agree with having a setting for more 'curated' suggestions, though when cutting suggestions it may be worth bearing in mind that some of them will be for common slips/errors (and probably shouldn't be learnt by rote).

**Comment by infodungeon** [comment]
Metadata tags for things like common mispellings might make it easier to parse the dictionary a bit more effectively.

**Comment by mkrnr** [comment]
Closing the ticket since it would be best implemented as a plugin.
